engine/api/workflow_run_test.go

```diff
@@ -1214,7 +961,7 @@ func Test_postWorkflowRunHandler(t *testing.T) {
 	assert.True(t, testFound, "should find 'test' in build parameters")
 }
 
-func waitCraftinWorkflow(t *testing.T, db gorp.SqlExecutor, id int64) error {
+func waitCraftinWorkflow(db gorp.SqlExecutor, id int64) error {
```

Comment:
please don't remove the `testing.T` pointer: the goal is to scope this function to test purpose